incident/api/serializers.py

```diff
@@ -89,6 +94,8 @@ class CategorySerializer(serializers.Serializer):
     methodology = serializers.CharField()
     page_symbol = serializers.CharField()
     plural_name = serializers.CharField()
+    slug = serializers.CharField()
```

Comment:
Added `slug` and `url`, the ticket has tasks to rename `page_symbol` but do we know what we want to rename it to?

Also there's a point about "Change symbol ID to a URL to the SVG instead" and while I can do that, I'm not clear on how that field can be used on the client-side. Looks like most of the svg icons are specified in CSS and generated with django? Am I missing something?

Reply:
@harrislapiroff  Do you have thoughts on `page_symbol` here? Did Accurat have a term more specific/meaningful they used when coming up with this concept?

Reply:
It looks like the symbols are being used on the database page, i.e. http://localhost:8000/all-incidents/ -- in the category graph and controls for selecting one in the filter sidebar. The code for that is centered around this, it seems like:

https://github.com/freedomofpress/pressfreedomtracker.us/blob/060333c917636a905bedb0633ceb9378f328ef96/client/charts/js/components/CategoryFilter.jsx#L32-L46

https://github.com/freedomofpress/pressfreedomtracker.us/blob/060333c917636a905bedb0633ceb9378f328ef96/client/charts/js/components/CategoryFilter.jsx#L210

It's not totally clear from the issue to me, but maybe the idea is to replace these SVG definitions in JS with ones directly on the category object that are drawn from the API?

Reply:
I think if we can provide a URL to the SVG associated with the category, that would be ideal, but it may be pretty tricky given that SVGs seem to get bundled by webpack with hash urls, [e.g.](https://pressfreedomtracker.us/static/bundles/15618b161cd6278fab09.svg).

If that's not easily possible I think we can remove the field? It looks like the bit that @chigby cited all references to the SVGs are encoded in the javascript, not pulled form the API, so it's not clear to me that we're using the data from the API anywhere.

Reply:
One other possibility, since these SVGs are probably fairly small, would be to actually send the raw SVG code. That should be retrievable from the filesystem, I think.

Reply:
I thought of another way, which is simply to use CSS to add the symbols the same way that it is being done for all the other category symbols used around the app. That way, it doesn't need to be hardcoded and it also doesn't need to be sent from the API. Thoughts?

Reply:
I think for the purposes of this PR let's leave it hard-coded and just remove the field from the API. We can revisit in the future, but I don't want to bikeshed too hard on this. (I'm concerned trying to add the symbols with CSS into an SVG will get dicey, so the right solution here doesn't seem immediately obvious to me.)

Reply:
Sure, happy to remove it and leave it hard-coded. Although I'm curious what you mean by "get dicey" in this case? I've used this pattern before a couple times and although it looks odd at first, the only scenario that I've seen issues with this is when the svg is transferred into some environment that isn't the DOM, such as downloading the svg or trying to render it to a canvas. If those are things we plan to do then I agree we should revisit this in the future.

Reply:
@dkaoster Okay, you've convinced me. Those definitely aren't things I expect us to be doing with this chart in particular. Go ahead and implement it the way you've proposed and then remove the SVG field from the API :)